docs/permissions/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -76,7 +76,7 @@ This enables decisions based on characteristics of the resource, but it's import
 Install the missing module:
 
 ```
-$ yarn workspace @internal/plugin-todo-list-backend add @backstage/plugin-permission-node
+$ yarn workspace @internal/plugin-todo-list-backend add @backstage/plugin-permission-node zod
```

Comment:
Since we're using `zod` for the schema now, we need to add it as a dep here. This is actually a bit tricky, since you need the _same version_ of zod across Backstage to avoid some very strange TS errors. It would be nice to find a way to encode this requirement, any thoughts @Rugvip perhaps?